homeassistant/components/yale_smart_alarm/alarm_control_panel.py

```diff
@@ -77,20 +77,22 @@ async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Set up the alarm entry."""
-
-    async_add_entities(
-        [YaleAlarmDevice(coordinator=hass.data[DOMAIN][entry.entry_id][COORDINATOR])]
-    )
+    coordinator: YaleDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id][
+        COORDINATOR
+    ]
+    async_add_entities([YaleAlarmDevice(coordinator)])
 
 
 class YaleAlarmDevice(CoordinatorEntity, AlarmControlPanelEntity):
     """Represent a Yale Smart Alarm."""
 
-    coordinator: YaleDataUpdateCoordinator
```

Comment:
```py
coordinator: YaleDataUpdateCoordinator
```

Is needed to specify the Coordinator class and should not be removed.

Reply:
Had to put this back or it doesn't work but the coordinator class is specified in the __init__